### PR TITLE
Sandbox improvements

### PR DESCRIPTION
**Pull request by winder**
* Let Indexer fetch genesis file on its own.
* Let Indexer write output to stdout so docker logs command works.
* Remove 'future' from dev mode network.
* Connect to psql shell when entering indexer-db container.

### CODE REVIEW

**Review by AlgoStephenAkiki — APPROVED** [review]

File: images/algod/DevModeNetwork.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
     "Genesis": {
-        "ConsensusProtocol": "future",
```
Comment (barnjamin) [inline images/algod/DevModeNetwork.json:3]: Why this change? 

Maybe misreading but would cause the proto version to be the latest from here:  https://github.com/algorand/go-algorand/blob/master/gen/generate.go#L77

Comment (winder) [reply images/algod/DevModeNetwork.json:3]: Future enables unreleased features in algod which increases the chance that Indexer would be incompatible.

For example, I was recently running the current Indexer against an old version of algod which has the"compact certificate" feature. It doesn't work without this change.

**Comment by tzaffi** [comment]
LGTM (away from keyboard)

File: images/algod/DevModeNetwork.json (hunk 1 above)
Comment (barnjamin) [reply images/algod/DevModeNetwork.json:3]: Fair enough,  it might confuse some exciting folks at first but def better for harder to debug issues. thanks!

**Review by barnjamin — APPROVED** [review]
